fairscale/optim/oss.py

```diff
@@ -517,20 +454,75 @@ def add_param_group(self, param_group: dict) -> None:
             # Update the bucketing strategy accordingly
             self._setup_flat_buffers()
 
+    @property
+    def _local_params(self) -> List[torch.Tensor]:
+        """ Iterable which goes through the parameters that this rank owns
+        """
```

Comment:
should this be on the previous line given this is a single line comment? 
nit: period at the end of the comment.